### ISSUE TITLE
Duplicate messages on master

### ISSUE THREAD

**Issue by bootstraponline** [Flank/flank#681]
```
---
gcloud:
  app: "../test_app/apks/app-debug.apk"
  test: "../test_app/apks/error-androidTest.apk"
flank:
  max-test-shards: -1
```

```
RunTests
  Uploading app-debug.apk   Uploading error-androidTest.apk ..


  Smart Flank cache hit: 0% (0 / 2)
  Shard times: 120s, 120s

  2 tests / 2 shards

  1 matrix ids created in 0m 1s
  https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2020-03-25_05-19-01.636000_MIOI/

PollMatrices
  0m  0s matrix-1h3w820q84crs NexusLowRes-28 VALIDATING
  0m  0s matrix-1h3w820q84crs NexusLowRes-28 VALIDATING
https://console.firebase.google.com/project/flank-open-source/testlab/histories/bh.bf178d418be9d33e/matrices/6081298694548031160
  0m  5s matrix-1h3w820q84crs NexusLowRes-28 PENDING
  0m  5s matrix-1h3w820q84crs NexusLowRes-28 PENDING
  0m 26s matrix-1h3w820q84crs NexusLowRes-28 Starting attempt 1.
  0m 26s matrix-1h3w820q84crs NexusLowRes-28 RUNNING
  0m 26s matrix-1h3w820q84crs NexusLowRes-28 Starting attempt 1.
  0m 26s matrix-1h3w820q84crs NexusLowRes-28 RUNNING
  1m 23s matrix-1h3w820q84crs NexusLowRes-28 Started logcat recording.
  1m 23s matrix-1h3w820q84crs NexusLowRes-28 Preparing device.
  1m 28s matrix-1h3w820q84crs NexusLowRes-28 Started logcat recording.
  1m 28s matrix-1h3w820q84crs NexusLowRes-28 Preparing device.
  1m 49s matrix-1h3w820q84crs NexusLowRes-28 Installing apps.
```

I think we may need to print more detailed information so that the logs don't appear to be duplicated. It looks like there's one log per shard.

**Comment by pawelpasterz** [comment]
I think we should keep one message per matrix. Since Flank supports server side sharding (for android) we are not able to fetch detailed info about single shard (or do we?). For ios it would remain as it is

**Comment by bootstraponline** [comment]
> I think we should keep one message per matrix. Since Flank supports server side sharding (for android) we are not able to fetch detailed info about single shard (or do we?). For ios it would remain as it is

I think the API might expose some additional details within a matrix we could print to the user that would differentiate the matrices. Maybe the execution id?

This ticket is a research task to see what the new API exposes and investigate if we can print more distinct and informative messages

**Comment by pawelpasterz** [comment]
Yeah, just after I wrote comment I've checked and there is info about single sharding. That's why I deleted it.

**Comment by pawelpasterz** [comment]
@jan-gogo I think this issue was solved with #779 . Do you think we can close it?
